oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreServiceConfigurationTest.java

```diff
@@ -115,6 +118,22 @@ public void fullGCEnabled() throws Exception {
         assertEquals(fullGCDocStore, config.fullGCEnabled());
     }
 
+    @Test
```

Comment:
Please consider also testing the logic when a path element appears in both the includes and excludes list.
Depending on what we decide the logic should be in this case (I propose that exclude takes priority), the test should reflect that.

Reply:
I believe both lists should be different, with even one common path, nothing gets fetched from DB.
cc @stefan-egli

Reply:
> I believe both lists should be different, with even one common path, nothing gets fetched from DB. cc @stefan-egli

my above is wrong, only common path is excluded not all.

Reply:
Check for invalid paths is added here: https://github.com/apache/jackrabbit-oak/pull/1551/commits/772c8559d4fdfa7e9fd8d251272f18606325ca04